.circleci/config.yml

```
@@ -313,7 +312,7 @@ jobs:
       - run:
           name: 🔎 run unit tests
           command: |
-              # unfortunately testthat does not and will not support returning a status
+              # unfortunately test that does not and will not support returning a status
```

```suggestion
              # unfortunately testthat does not and will not support returning a status
```
https://testthat.r-lib.org/